runbotics/runbotics-common/src/model/api/queue-message.model.ts

```diff
@@ -0,0 +1,40 @@
+import { IProcess } from "./process.model";
+import { ProcessInput } from "./queue.model";
+import { WsMessage } from "./websocket.model";
+
+type JobId = string | number;
+
+export interface BaseQueueMessageBody {
+    initialDate: string;
+    updateDate: string;
+    processQueueId: JobId;
+    input: ProcessInput;
+    processId: IProcess["id"];
+}
+
+export interface GeneralQueueMessageBody extends BaseQueueMessageBody {
+    type: "ACTIVE" | "REMOVE" | "TIMEOUT" | "FAILED";
```

Comment:
adjust types to be more descriptive about runbotics app processes rather than inform about stages of the queue

Reply:
plus imo make it enum as you use it in the code at scheduler